src/lib/cli/format.js

```diff
@@ -122,14 +122,27 @@ export function requoteArgs( args: Array<string> ): Array<string> {
 			return arg.replace( /^--(.*)=(.*)$/, '--$1="$2"' );
 		}
 
-		if ( arg.includes( ' ' ) ) {
+		if ( arg.includes( ' ' ) && ! isJsonObject( arg ) ) {
 			return `"${ arg }"`;
 		}
 
 		return arg;
 	} );
 }
 
+export function isJsonObject( str: string ): boolean {
```

Comment:
This is incredibly minor, but I don't love variable / argument names like `str`. I would vote for following the `JSON.parse` naming of `text`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#parameters

But feel free to ignore me. Just personal preference.